lib/functions/aeminitiateupload.js

```diff
@@ -116,7 +118,14 @@ class AEMInitiateUpload extends AsyncGeneratorFunction {
 
             // yield one or more smaller transfer parts if the source accepts ranges and the content length is large enough
             const files = initiateResponse.files;
-            const completeUrl = new URL(initiateResponse.completeURI, folderUrl);
+
+            let completeURI = initiateResponse.completeURI;
+            if (this.options.supportSpecialCharacters) {
```

Comment:
Looks like `this.options` could be null/undefined. 

```suggestion
            if (this.options && this.options.supportSpecialCharacters) {
```

Reply:
Good catch - I included this check.

Taylor